curve/utils.go

```diff
@@ -1,8 +1,9 @@
-package starknetgo
+package curve
```

Comment:
I think we can merge utils into curve. The StarkCurve type is defined in curve, along with a collection of StarkCurve methods. Utils just defines some more methods on StarkCurve.